src/style-installer.js

```diff
@@ -283,6 +270,25 @@ export function makeBodyVisible(doc, opt_waitForServices) {
   }
 }
 
+/**
+ * Set the document's body opacity to 1. Called in error cases.
+ * @param {!Document} doc The document who's body we should make visible.
+ */
+export function makeBodyVisibleRecovery(doc) {
+  dev().assert(doc.defaultView, 'Passed in document must have a defaultView');
+  const win = /** @type {!Window} */ (doc.defaultView);
+  if (win[bodyVisibleSentinel]) {
+    return;
+  }
+
+  win[bodyVisibleSentinel] = true;
+  setStyles(dev().assertElement(doc.body), {
```

Comment:
This is simple chunk of code, but I'd still prefer if it was in its own function called from both `makeBodyVisible` and `makeBodyVisibleRecovery`.

Reply:
Done.